src/common/BO/loginBO.ts

```diff
@@ -1,11 +1,10 @@
-import testContext from '@utils/testContext';
+//import testContext from '@utils/testContext';
 
-import loginPage from '@pages/BO/login';
+let loginPage = require('@pages/BO/login');
 import dashboardPage from '@pages/BO/dashboard';
 
 import {expect} from '@playwright/test';
-import type {Page} from 'playwright';
-import type {TestInfo} from '@playwright/test';
```

Comment:
Les imports ne sont pas ordonnés, et d'un dev à l'autre ça peut causer des changements inutiles / obfusquer de réelles modifications. Pour cela ma reco c'est de configurer eslint pour cela https://eslint.org/docs/latest/rules/sort-imports

Reply:
Don't worry. ESLint is :red_circle: